saleor/graphql/core/fields.py

```diff
@@ -10,19 +11,48 @@
 from .connection import FILTERS_NAME, FILTERSET_CLASS
 
 
-class PermissionsField(graphene.Field):
-    def __init__(self, *args, **kwargs):
+class FieldCost(TypedDict, total=False):
+    complexity: int
+    multipliers: List[str]
+    use_multipliers: bool
+
+
+class CostField(graphene.Field):
+    def __init__(
+        self,
+        *args,
+        cost: Optional[FieldCost],
+        sensitive: bool = False,
+        **kwargs,
+    ):
+        self.sensitive = sensitive
+        self.cost = cost
+        super().__init__(*args, **kwargs)
+
+
+SensitiveField = partial(CostField, cost=None, sensitive=True)
```

Comment:
It would be nice to add docstring to all new classes. E.g. here we should explain what types of fields we consider sensitive or what impact does it have to mark a field as such.

Reply:
Agree! I will prepare a definition for sensitive and cost fields and include it in docstrings